CMakeLists.txt

```
@@ -14,6 +14,7 @@ project(mamba)
 option(BUILD_SHARED "Build shared libmamba library" OFF)
 option(BUILD_STATIC "Build static libmamba library with static linkage to its dependencies" OFF)
 option(BUILD_LIBMAMBA "Build libmamba library" OFF)
+option(BUILD_LIBMAMBA_SPDLOG_TESTS "Build libmamba-spdlog library tests" OFF)
```

I think that we would need this to be able to distribute `libmamba-spdlog` independently.

```suggestion
option(BUILD_LIBMAMBA_SPDLOG "Build libmamba-spdlog library" OFF)
option(BUILD_LIBMAMBA_SPDLOG_TESTS "Build libmamba-spdlog library tests" OFF)
```